handbook/business-operations/people-operations.md

```diff
@@ -242,6 +242,8 @@ There are some defaults that we use for these agreements:
 
 Then hit send!  After all of the signatures are there, the completed document will automatically be uploaded to the appropriate Google Drive folder, and a Slack message will appear in the `#help-classified` channel.
 
+Once the agreement is signed and in the appropriate folder, you will then proceed to ["Steps after an offer is accepted"](https://fleetdm.com/handbook/business-operations/people-operations#steps-after-an-offer-is-accepted), and complete all relevant activities.
```

Comment:
We don't follow these steps for consultants- they don't end up in Fleeties or the equity plan, normally don't get email addresses.

I think what we need instead is the step of adding them here:
<img width="247" alt="image" src="https://github.com/fleetdm/fleet/assets/618009/fe4b9dd8-2a0b-4035-9a95-0fc7fa424ee2">

So that they get addressed by the weekly ritual of reporting consultant hours to the CEO: https://github.com/fleetdm/fleet/blob/73559b8298ee3d13d119e6319d69c5e7af55e680/handbook/business-operations/people-operations.md#informing-managers-about-hours-worked

Reply:
I like that. Will make that change. We need a consultant onboarding issue too, going through adding to slack, access to any other systems or documents, etc.